index.js

```
@@ -155,24 +157,38 @@ module.exports = function createPlugin(app) {
+            // new engine
+            if (!engine) {
+              engine = {
                 path: v.path,
                 runTime: 0,
                 runTimeTrip: 0,
-                time: new Date().toISOString(),
               };
-              engines.paths.push(pathObject);
-              scheduleDebouncedWrite();
+              engines.paths.push(engine);
             }
```

ESLint

[error] Closing curly brace does not appear on the same line as the subsequent block.

(brace-style)